### ISSUE TITLE
Better text in logbook for trigger state

### ISSUE THREAD

**Issue by tomer-w** [nielsfaber/alarmo#895]
**Preface:** Alarmo is the best HA integration I know of. I've been using it for years, and it just keeps working and improving. Amazing work!!

**Idea**: When examining the HA Logbook page for the Alarmo entity, I mostly see excellent data (green in my snip below), which can help me understand what was going on. The only exception is the triggered state, which includes no information to help investigate the issue (red in my snip below).

![image](https://github.com/nielsfaber/alarmo/assets/57483589/1269a7f1-30fd-4725-aea8-6b34007bed73)

### Checklist
- [X] I checked for similar existing requests (both open and closed) before posting.
- [X] My request is generic, other users may benefit from it too.

### Proposal
It will be great to have all entities that caused the state change show up. This is already supported in the notification feature, but that is an opt-in feature that users need to understand how to configure and how not to lose it later when investigating something. Having it in the logbook makes it way easier to handle.

### Additional info
Instead of: Home Alarm changed to Triggered
Should be: Home Alarm changed to Triggered. Cause {{open_sensors}}

**Comment by nielsfaber** [comment]
I fully agree with you this information should be displayed here.
I can also think of other scenarios, like the alarm failing to arm due to some open sensor.
The question is: how?

As far as I know, the logbook component of HA decides on its own which information is displayed.
I am not aware of any provision I can make in the code of Alarmo to make more information show up.
There is no or very little documentation available for developers to improve interaction with the logbook.
I believe the correct term for this extra information is _context_: https://data.home-assistant.io/docs/context

So in summary, if you or anyone else can give some pointers on how to add provisions for this, I'd be happy to do so. 
At the moment I have no idea if or how your request can be fulfilled.
